showcase/src/assets/translations/en.json

```diff
@@ -109,6 +109,28 @@
         "SIMPLE_HORIZONTAL_SLIDER": "Simple Horizontal Slider",
         "SIMPLE_VERTICAL_SLIDER": "Simple Vertical Slider",
         "VERTICAL_RANGE_SLIDER": "Vertical Range Slider"
+      },
+      "TYPOGRAPHY": {
```

Comment:
Could you order these translations in alphabetical order so they are easier to locate?